logging-modules/log4j/pom.xml

```diff
@@ -43,13 +44,20 @@
             <artifactId>disruptor</artifactId>
             <version>${disruptor.version}</version>
         </dependency>
+        <!-- Webflux for reactive logging example -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-webflux</artifactId>
+            <version>${spring-boot.version}</version>
+        </dependency>
     </dependencies>
 
     <properties>
         <log4j.version>1.2.17</log4j.version>
         <log4j-api.version>2.7</log4j-api.version>
         <log4j-core.version>2.7</log4j-core.version>
         <disruptor.version>3.3.6</disruptor.version>
+        <spring-boot.version>2.0.5.RELEASE</spring-boot.version>
```

Comment:
Let's use the latest version of Spring here (2.1.0)

Reply:
Done.